(MAINT) fix pathing to inventory.yaml and README instructions

ThoughtCrhyme opened the pull request.

Commit:
(MAINT) fix pathing to inventory.yaml and README instructions

Commit:
Fix cert signing on windows and test against linux and windows

ThoughtCrhyme commented on plans/agents.pp:
```diff
@@ -4,13 +4,15 @@
   # set the server 
   $server_string = $server[0].name
   run_task('puppet_conf', $agents, action => 'set', section => 'main', setting => 'server', value => "${server_string}")
+  run_command("powershell.exe -NoProfile -Nologo -Command 'Remove-Item -Path /ProgramData/PuppetLabs/puppet/etc/ssl -Force -Recurse'", $windows_agents)
```
I have to be honest, I don't really know why nuking the ssl directory works, but it does :/

tphoney replied:
yeah that's disappointing :( likely to do with how puppet_agent is working on windows.

Commit:
(MAINT) add a way to teardown all provisioned nodes.

Depends on a change submitted to the provision project.

tphoney reviewed (changes requested):
we should add the teardown task to the readme

tphoney commented on plans/tester.pp:
```diff
@@ -3,10 +3,16 @@
   $server = get_targets('*').filter |$n| { $n.vars['role'] == 'pe' }
   $agent_linux = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent_linux/ }
-
+  $agent_windows = get_targets('*').filter |$n| { $n.vars['role'] =~ /agent_windows/ }
```
can you make the regex  /agent/ then we can just run against all agents, rather than differentiating

ThoughtCrhyme replied:
done

Commit:
(MAINT) iterate over each node for teardown in teardown plan

This commit allows us to remove all the provisioned nodes without touching
the provision project.
